HousingManagementSystemApi/Models/UniversalHousingAddress.cs

```diff
@@ -0,0 +1,16 @@
+namespace HousingManagementSystemApi.Models
+{
+    public class UniversalHousingAddress
+    {
+        public string PropertyReference { get; set; }
+
+        public string Uprn { get; set; }
+
+        public string PostDesig { get; set; }
```

Comment:
what is this again?

Reply:
`PostDesig` is the Universal Housing database column name for door number. I _think_ `PostDesig` may be a shortening of "Postal Designation".